src/useq/_grid.py

```diff
@@ -189,35 +164,41 @@ def _step_size(self, fov_width: float, fov_height: float) -> Tuple[float, float]
         return dx, dy
 
 
-class GridFromCorners(_GridPlan):
+class GridFromEdges(_GridPlan):
     """Define grid positions from two corners.
 
     Attributes
     ----------
-    corner1 : Coordinate
-        First bounding coordinate (e.g. "top left"). The position is considered
-        to be in the center of the image.
-    corner2 : Coordinate
-        Second bounding coordinate (e.g. "bottom right"). The position is considered
-        to be in the center of the image.
+    top : float
+        y value of the top left bounding coordinate
+    left : float
+        x value of the top left bounding coordinate
+    bottom : float
+        y value of the bottom right bounding coordinate
+    right : float
+        x value of the bottom right bounding coordinate
```

Comment:
fix all of these... the top doesn't care whether it's left/right.  the lefty doesn't care whether it's top/bottom, etc... it's just "top bound", "bottom bound", etc...